GVRf/Framework/src/org/gearvrf/GVRContext.java

```diff
@@ -341,11 +362,12 @@ public GVRMesh createQuad(float width, float height) {
                 width * 0.5f, height * -0.5f, 0.0f };
         mesh.setVertices(vertices);
 
-        final float[] normals = { 0.0f, 0.0f, 1.0f, 0.0f, 0.0f, 1.0f, 0.0f, 0.0f,
-                1.0f, 0.0f, 0.0f, 1.0f };
+        final float[] normals = { 0.0f, 0.0f, 1.0f, 0.0f, 0.0f, 1.0f, 0.0f,
+                0.0f, 1.0f, 0.0f, 0.0f, 1.0f };
```

Comment:
Just a minor issue. I know this is just for formatting. But I think I saw it from another pull request.. Not sure if it will lead to code conflict.